packages/cli/lib/LocalDevManager.js

```diff
@@ -159,14 +160,10 @@ class LocalDevManager {
       indent: 1,
       category: 'header',
     });
-    const projectDetailUrl = getProjectDetailUrl(
-      this.projectConfig.name,
-      this.targetAccountId
-    );
     this.spinnies.add('viewInHubSpotLink', {
       text: uiLink(
         i18n(`${i18nKey}.header.viewInHubSpotLink`),
-        projectDetailUrl,
+        `${this.devServerPath}/hs/project`,
```

Comment:
What happens if the `this.devServerPath` remains `null`? Is that possible?

Reply:
done in [fe1c962](https://github.com/HubSpot/hubspot-cli/pull/850/commits/fe1c9622912af7e681d4f05cb2e3484b5849afd9)